client/app/dedicatedCloud/dedicatedCloud.controller.js

```diff
@@ -37,6 +38,36 @@ angular.module("App").controller("DedicatedCloudCtrl", [
 
         $scope.dedicatedCloud = {};
 
+        function showNotificationIfRequired (notification) {
+            Notification.checkIfStopNotification(notification, $stateParams.productId)
+                .then((stopNotification) => {
+                    $scope.notifications[notification] = !stopNotification;
+                })
+                .catch((error) => {
+                    $scope.notifications[notification] = true;
+                    $log.error(error);
+                });
+        }
+
+        function handleCancelConfirmation () {
```

Comment:
I think you are not up to date with `develop`. I think I have remove this part during refactoring some modals into dedicatedCloud section.